bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/PairVLQ_x53x53_tHtH_narrow/PairVLQ_x53x53_tHtH_narrow_RH_MX1100_MH400/PairVLQ_x53x53_tHtH_narrow_RH_MX1100_MH400_run_card.dat

```diff
@@ -266,6 +266,5 @@
 #          meaningful ONLY if plotted taking matchscale              *
 #          reweighting into account!                                 *
 #*********************************************************************
-   T  = pdfwgt      ! Enable systematics studies
-#
-#**************************************
+   T  = use_syst      ! Enable systematics studies
+
```

Comment:
Hi @menglu21 I checked with the theorists on this since there was a bit of confusion on my part here but this is indeed correct that the model is 5f and so I made the change of 5 = maxjetflavor. They did have a concern about using the correct lhapdf index to avoid loading PDFs in the 5FNS. I set the lhapdf to reweight_PDF here. Could you confirm that this is the correct method please.

Reply:
Hi @farrah-simpson by using "$DEFAULT_PDF_MEMBERS = reweight_PDF", the 5f or 4f will be used according to the "p" definition autoly. btw, the maxjetflavor is still 4, have you committed your change

Reply:
@menglu21 thank you for this clarification. I will commit my changes shortly. I am making one more change to my coupling parameters.